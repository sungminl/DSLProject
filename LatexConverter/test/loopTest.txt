&a = 1

LOOP &a FROM 1 - 1 TO 5
{
MATRIX
ROW: 2, COL: 4(&a)
{
2 * &a|
&a / 2|
}
}
