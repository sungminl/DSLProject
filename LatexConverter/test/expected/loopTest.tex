\documentclass[]{article}
\usepackage{amsmath}
\usepackage{tikz}
\usetikzlibrary{graphs,automata}
\title{Document}
\begin{document}
\[\begin{bmatrix}
0.0 & 0.0 & 0.0 & 0.0\\
0.0 & 0.0 & 0.0 & 0.0\\
\end{bmatrix}\]
\[\begin{bmatrix}
2.0 & 2.0 & 2.0 & 2.0\\
0.5 & 0.5 & 0.5 & 0.5\\
\end{bmatrix}\]
\[\begin{bmatrix}
4.0 & 4.0 & 4.0 & 4.0\\
1.0 & 1.0 & 1.0 & 1.0\\
\end{bmatrix}\]
\[\begin{bmatrix}
6.0 & 6.0 & 6.0 & 6.0\\
1.5 & 1.5 & 1.5 & 1.5\\
\end{bmatrix}\]
\[\begin{bmatrix}
8.0 & 8.0 & 8.0 & 8.0\\
2.0 & 2.0 & 2.0 & 2.0\\
\end{bmatrix}\]
\end{document}
